app/src/androidTest/java/com/android/sample/screen/mainscreen/CalendarScreenTest.kt

```diff
@@ -121,7 +121,7 @@ class CalendarScreenTest {
     // Find all nodes with the testTag "daySection"
     val dayNodes = composeTestRule.onAllNodesWithTag("daySection", useUnmergedTree = true)
 
-    // Assert that there are 7 day sections displayed
+    // Assert that there are 3 day sections displayed
     assert(3 < dayNodes.fetchSemanticsNodes().size)
   }
```

Comment:
Maybe consider adding comments explaining why you chose 3 day sections

Reply:
already explained in the #75 PR